pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                               get_cmd=sintlength_cmd + '?',
+                               set_cmd=sintlength_cmd + ' {}',
+                               vals=vals.Numbers(1, 2048)
+                               )
+            swintstat_cmd = 'qutech:wint{}:status'.format(ch_pair)
+            self.add_parameter('ch_pair{}_wint_status'.format(ch_pair),
+                               units='#',
+                               label=('weighted integral status of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=swintstat_cmd + '?'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat00_cmd = 'qutech:rotmat{}:rotmat00'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat00'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 00 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat00_cmd + '?',
+                               set_cmd=srotmat00_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat01_cmd = 'qutech:rotmat{}:rotmat01'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat01'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 00 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat01_cmd + '?',
+                               set_cmd=srotmat01_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat10_cmd = 'qutech:rotmat{}:rotmat10'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat10'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 10 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat10_cmd + '?',
+                               set_cmd=srotmat10_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat11_cmd = 'qutech:rotmat{}:rotmat11'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat11'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 11 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat11_cmd + '?',
+                               set_cmd=srotmat11_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            '''
+            TV mode parameters
```

Comment:
In the existing drivers `TV mode` is called `Integration averaging` or `int_avg`. To be consistent this should be renamed to integration averaging. (also in everywhere following this. 

Of course this does not require changing the VISA commands.